scripts/travis/build.sh

```diff
@@ -49,6 +49,17 @@ else
   EXTRA_ARGS=''
 fi
 
+# For PR builds, Skip module-specific tests if its module dependencies haven't been touched
```

Comment:
We would still run all tests for the release build, correct?

Reply:
This if-statement means that it'll only do this conditional skipping if the build is from a PR. This means all tests will be run for tag pushes (the release build) and builds on master.

However, this means a PR that's doing a major release can still have tests skipped. Not sure if that's ideal, though.

We can change this behavior to whatever we want. What do you think would be ideal?

Reply:
I think we do builds for the following steps:
- New PR, new commits on a PR: Ok to skip some tests that are not affected.
- PR merged to master: We currently don't tests for this, right? Should we run now? I'm on the fence for this.
- tag pushes/releases: We should run all tests for this.

Reply:
Just checked, for the second item, PR merged to master, we run all tests. We should keep it as is. 

so, only change should be the tests run for PRs